compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -73,7 +73,7 @@ pub(crate) struct CrateMetadata {
     /// The primary crate data - binary metadata blob.
     blob: MetadataBlob,
 
-    // --- Some data pre-decoded from the metadata blob, usually for performance ---
+    /// --- Some data pre-decoded from the metadata blob, usually for performance ---
```

Comment:
I think this was meant as section header, not as documentation for the `root` field.